### DISCUSSION TITLE
How to enable auto complete for the native and 3rd party Java libraries?

### DISCUSSION THREAD

**Discussion by phommata** [nextgenhealthcare/connect#6036] [Q&A]
JavaScript and the Mirth libraries have auto complete supported. How can I get them enabled for the native Java and 3rd party libraries?

**Comment by pacmano1** [comment]
I phrased it poorly in your other post.   This is not a feature today.  Create a feature request, there is a template to follow for it.

**Reply by phommata** [reply]
Where is the template? I searched for `template` in discussions and I do not see it in the README.

**Reply by pacmano1** [reply]
![image](https://github.com/nextgenhealthcare/connect/assets/44065187/7f320ac9-a130-4fb1-9270-4af66393b50b)
![image](https://github.com/nextgenhealthcare/connect/assets/44065187/f21aaa8f-1e24-4e72-96b3-6d0e2dca33e5)
![image](https://github.com/nextgenhealthcare/connect/assets/44065187/039ad825-2485-40b9-a380-83f5ecce701f)

**Reply by phommata** [reply]
Thank you very much for your guidance. I have successfully created a feature request under new issues using the template following your screenshots. https://github.com/nextgenhealthcare/connect/issues/6040
